packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -89,6 +91,22 @@ class _CodeViewState extends State<CodeView>
     textController = verticalController.addAndGet();
     horizontalController = ScrollController();
 
+    if (widget.initialPosition != null) {
```

Comment:
This block is duplicated directly below. delete one of them.

Reply:
Bad merge resolution I guess. Done.